tests/integration/s2n_handshake_test_gnutls.py

```diff
@@ -36,8 +36,8 @@ def try_gnutls_handshake(endpoint, port, priority_str):
     s2nd.stdout.readline()
 
     # Fire up gnutls-cli, use insecure since s2nd is using a dummy cert
-    gnutls_cli = subprocess.Popen(["gnutls-cli", "--priority=" + priority_str,"--insecure", "-p " + str(port), str(endpoint)], 
-                                   stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.DEVNULL)
+    gnutls_cli = subprocess.Popen(["gnutls-cli", "--priority=" + priority_str,"--insecure", "--recordsize=" + "512",
```

Comment:
Thank you for the insights,
I will check 1.  the server ignores the MFL extension case and get back to you. 
2. 3. will update the integration test.